sources.go

```diff
@@ -83,6 +83,31 @@ func ListFiles(path string) *Pipe {
 	return Slice(fileNames)
 }
 
+// FindFiles creates a pipe containing the files inside a directory and all its subdirectories
```

Comment:
Functions are in alphabetical order in this file, just to keep some semblance of organization—so this should actually go before `ListFiles`.